docs/howto/conditional.adoc

```diff
@@ -0,0 +1,65 @@
+= How to perform actions based on previous command results
+
+qDup commands under a `then` will run if the parent command ran successfully and will run on the output of the parent command. For example:
+```yaml
+#...
+- sh: date
+  then:
+  - regex: Fri
+    then:
+    - log: "Pizza Friday!"
+#...
+```
+The `regex` command runs with input from the parent `sh` command and then `log` command runs if the `regex` successfully matched. If we ran with `check-exit-code` then the `regex` would only run if the `date` exit code was zero.
```

Comment:
Please say that `regex` runs on every **line** of output and `log`will therefore execute on every line. Btw. is there a way to run regex incl. the newlines?

Reply:
If `regex` is in a `then` it runs on the entire output after the parent command is finished. If it is in a `watch` it works line by line because `watch` is line by line while the watched command runs. 
For newlines, do you mean having the `.` match the newline character? There is the `(?s)` embedded dot-all that works at the start of the pattern.
https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/util/regex/Pattern.html#DOTALL

Reply:
Thanks, I confused it with `watch`. In that case please clarify that in the docs (`with input from the parent sh` -> `with complete output (all lines from stdout) of the parent sh as input` or any better wording of that...)